querydsl-sql/src/main/java/com/querydsl/sql/SQLSerializer.java

```diff
@@ -162,22 +149,22 @@ public void handle(String template, Object... args) {
     }
 
     public final SQLSerializer handleSelect(final String sep, final List<? extends Expression<?>> expressions) {
-        if (inSubquery) {
-            Set<String> names = new HashSet<>();
-            List<Expression<?>> replacements = new ArrayList<>();
-            for (Expression<?> expr : expressions) {
-                if (expr instanceof Path) {
-                    String name = ColumnMetadata.getName((Path<?>) expr);
-                    if (!names.add(name.toLowerCase())) {
-                        expr = ExpressionUtils.as(expr, "col__" + name + replacements.size());
-                    }
-                }
-                replacements.add(expr);
-            }
-            return handle(sep, replacements);
-        } else {
-            return handle(sep, expressions);
-        }
+        if (!inSubquery) {
```

Comment:
Negative conditionals are not considered more readable.

Reply:
Negative conditional statements are not easier to read.
They just make the code easier to read by reducing its depth.

You might want to check out
https://go.dev/doc/effective_go#if
https://stackoverflow.com/questions/2928556/programming-style-should-you-return-early-if-a-guard-condition-is-not-satisfied